konnect-runtime-setup.sh

```diff
@@ -361,23 +271,15 @@ main() {
     # validating required variables
     check_variables
 
-    # login and acquire the session
-    login
-
-    # get control plane data
-    get_control_plane
-
-    # retrieve certificates, keys for runtime
-    generate_certificates
+    # verify validity of the passed in key and certificate
+    verify_certificates
 
     # download kong docker image
     download_kongee_image
 
     echo "Ready to launch"
     run_kong
     echo "Enjoy the flight!"
-
-    cleanup
```

Comment:
I guess we still want to cleanup in the end.

Reply:
So the only generated files are cluster.key and cluster.crt. I can't delete them here because the docker command is running and will require those files. Not sure how to wait until the docker command finishes.

Reply:
you are right. let's remove the cleanup in the end.